generator/default_package.go

```diff
@@ -27,6 +27,8 @@ type DefaultPackage struct {
 	// Import path of the package, and the location on disk of the package.
 	PackagePath string
 
+	Source string
```

Comment:
go doc. What is source path? The current working directory?

Reply:
It's the location of the package on disk. I've added godoc.